test/api/layout/execute_with_estimation_test.cpp

```diff
@@ -37,8 +37,23 @@ TEST(execute_estimation_test, few_ubs)
     testing::internal::CaptureStderr();
     chopper::layout::execute(layout_parser);
 
-    EXPECT_EQ(testing::internal::GetCapturedStdout(), "T_Max\tC_{T_Max}\trelative expected HIBF query cost\n64\t1.00"
-                                                      "\t1.00\n#Best t_max (regarding expected query runtime):64\n");
+    EXPECT_EQ(testing::internal::GetCapturedStdout(),
+R"expected_cout(## Parameter: number of user bins = 8
+## Parameter: number of hash functions = 2
+## Parameter: false positive rate = 0.05
+## Note: X-IBF = An IBF with X number of bins.
+## Note: X-HIBF = An HIBF with tmax = X, e.g a maximum of X technical bins on each level.
+## Column tmax : The maximum number of technical bin on each level
+## Column c_tmax : The technical extra cost of querying an tmax-IBF, compared to 64-IBF
+## Column l_tmax : The estimated query cost for an tmax-HIBF, compared to an 64-HIBF
+## Column m_tmax : The estimated memory consumption for an tmax-HIBF, compared to an 64-HIBF
+## Column (l*m)_tmax : Computed by l_tmax * m_tmax
+## Column size : The expected total size of an tmax-HIBF
+#tmax	c_tmax	l_tmax	m_tmax	(l*m)_tmax	size
+64	1.00	1.00	1.00	1.00	17 KiB
```

Comment:
Maybe `to_formatted_BF_size` should return the size without space? I.e. `17KiB` instead of `17 KiB`

Reply:
Ah yes, I also thought of that and then forgot about it again.

Either that or always output KiB ?

Reply:
I like that it uses an appropriate unit. 
`1TiB == 1073741824KiB`